### PR TITLE
Feat triple masking v2 ledger status

### PR DESCRIPTION
**Pull request by harshadptl**
* **make sure that you have executed all the following process and no errors occur**
  - [x] make fmt
  - [x] make lint
  - [x] make test

* **The major changes of this PR**

* **Extra documentations**

### CODE REVIEW

File: src/ledger/src/store/mod.rs [hunk 1]
```diff
@@ -272,11 +277,12 @@ impl LedgerState {
             self.utxo_map.clear(inp_sid.0 as usize).c(d!())?;
         }
 
+        let backup_next_txn_sid = self.status.next_txn.0;
```
Comment (Jedi2002) [inline src/ledger/src/store/mod.rs:280]: Is this value different from self.status.next_atxo

File: src/ledger/src/store/mod.rs [hunk 2]
```diff
@@ -298,8 +305,26 @@ impl LedgerState {
             }
             self.nullifier_set.set(&d, Some("".to_string()));
         }
-        for abar in output_abars.iter() {
-            self.add_abar(&abar).c(d!())?;
+
+        let mut txn_sid = TxnSID(backup_next_txn_sid);
```
Comment (Jedi2002) [inline src/ledger/src/store/mod.rs:309]: Is this value different from self.status.next_atxo

**Review by Jedi2002 — APPROVED** [review]
Looks find to me , added a minor comment

File: src/ledger/src/store/mod.rs (hunk 2 above)
Comment (harshadptl) [reply src/ledger/src/store/mod.rs:309]: Ya, I am creating a backup for txn_sid before the block effects are applied. This is to apply anon outputs separately compared to the TxOutput changes.

**Pull request merged**